### PR TITLE
docs(api): remove redundant new keyword

### PR DESCRIPTION
**Pull request by jamesgeorge007**
Follow up of #3545 

`createFsFromVolume()` is a function and should be invoked directly.

### CODE REVIEW

File: src/content/contribute/writing-a-loader.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ contributors:
   - michael-ciniawsky
   - byzyk
   - anikethsaha
+  - jamesgeorge007
```
Comment (montogeek) [inline src/content/contribute/writing-a-loader.md]: Why this change?

Comment (jamesgeorge007) [reply src/content/contribute/writing-a-loader.md]: In regard to https://github.com/webpack/webpack.js.org/pull/3493. The list wasn't updated as part of that PR :sweat_smile:

**Comment by montogeek** [comment]
Thanks!
